config/initializers/octokit.rb

```diff
@@ -0,0 +1,32 @@
+if Rails.env.production?
+  Octokit.configure do |config|
+    config.login = ENV.fetch('GH_USERNAME')
+    config.password = ENV.fetch('GH_PASSWORD')
+  end
+end
+
+class GithubFakeAdapter
+  def initialize(_post); end
+
+  def create_pr
+    Rails.logger.debug('Create a PR')
+  end
+
+  def update_pr
+    Rails.logger.debug('Update the PR')
+  end
+
+  def close_pr
+    Rails.logger.debug('Close the PR')
+  end
+end
+
+module Github
+  def self.adapter
+    if Rails.env.production?
+      GithubAdapter
+    else
+      GithubFakeAdapter
+    end
+  end
+end
```

Comment:
That's the thing with this all being in the same place, I don't think they have enough value to have their own files, and it would probably be harder to understand what they are for if they are in different files. That was my reasoning.